### PR TITLE
feat(video): Adds ability to enable & retrieve HEVC profiles

### PR DESCRIPTION
**Pull request by asciidisco**
I urge @trummerjo & @liberty-developer to have a look on this, as I have no experience with HEVC profiles & don't know if all of them should be enabled.

### CODE REVIEW

File: resources/lib/MSL.py [hunk 1]
```diff
@@ -440,9 +438,9 @@ def __generate_msl_request_data(self, data):
         }
 
         # Serialize the given Data
-        serialized_data = json.dumps(data)
-        serialized_data = serialized_data.replace('"', '\\"')
-        serialized_data = '[{},{"headers":{},"path":"/cbp/cadmium-13","payload":{"data":"' + serialized_data + '"},"query":""}]\n'
+       serialized_data = json.dumps(data)
+       serialized_data = serialized_data.replace('"', '\\"')
+     serialized_data = '[{},{"headers":{},"path":"/cbp/cadmium-13'"payload":{"data":"' + serialized_data + '"},"query":""}]\n'
```
Comment (trummerjo) [inline resources/lib/MSL.py]: `"path":"/cbp/cadmium-13", "payload"`?

**Comment by asciidisco** [comment]
@trummerjo Silly me committed to fast. Issues have been resolved.

**Comment by trummerjo** [comment]
regarding the profiles: i dont know either which ones are needed. For now it should be finde to enable all of them

**Comment by im85288** [comment]
Just wanted to provide my feedback having just tested this on android shield tv. 

Initially I thought it was not working correctly as the resolution was always either 1080p or 720p, I then went into InputStream settings and changed the stream selection to manual. With this activated playing an episode initially started in a really low resolution but I can go into the video settings and then select the highest quality 4k stream. Once that is selected it plays flawlessly. I do however have one question and that is with regards HDR. I used an episode of Glow as a trial and in the native Netflix it plays with HDR but in kodi it is missing. Is that something that may be supported in the future. 

I think once auto select of the highest 4k stream can be done and HDR is added it's feature complete.

**Comment by trummerjo** [comment]
First of all: Great to hear that it is working with 4K streams on Android!
The problem that the HDR stream ist not displayed could have more reasons:
1) We generate a wrong ESN for the netflix API and netflix does not provide the HDR Streams in the manifest
2) We don't have the right profiles and the netflix API does not providehe HDR Streams in the manifest

I would bet it is reason 2).

The problem is: It is very hard to find out the correct profiles. The profiles now included are taken from Microsoft's Browser Edge. Android may have diffrent ones. For now I dont have an idea how to find them out. Decomiling the APK and digging in the code may be the best approach.

**Comment by asciidisco** [comment]
@im85288 Thanks for testing it (and being extremely quick in doing so).
I´ll try to investigate (or poke @peak3d to give it a shot) why the best possible profile isn't chosen. In the meantime, to make this feature available, we should add a note to the README about this, so that people know that they should use the manual stream selection.

**Comment by asciidisco** [comment]
Updated README & corresponding issue

**Comment by im85288** [comment]
No problem at all thanks for adding it, very cool to have 4k streams working.

I did some googling and it looks like HDR is coming to Microsoft Edge with the October 17th release of windows 10 fall creators update. The only link I could find was in german: https://translate.google.co.uk/translate?hl=en&sl=de&u=https://www.heise.de/newsticker/meldung/Nvidia-Netflix-Streaming-in-4K-und-HDR-moeglich-ab-Windows-10-Fall-Creators-Update-3842435.html&prev=search
Hopefully when that comes out the new profiles can be taken from the edge browser which would save digging into the APK

**Pull request merged**